frontend/src/libraries/wamp/types.ts

```diff
@@ -1,9 +1,11 @@
+import * as RPC from "../../../../backend/src/rpc-types";
```

Comment:
It is quite concerning that we directly depend on backend source code in the frontend. At the very least, I would like to suggest to have a common package in the root of the project (ideally, added to both backend/frontend as a regular package [package.json])

P.S. As I have already mentioned, this cross-project dependency will break Docker images